tests/kafkatest/services/connect.py

```diff
@@ -76,6 +78,41 @@ def set_configs(self, config_template_func, connector_config_templates=None):
         self.config_template_func = config_template_func
         self.connector_config_templates = connector_config_templates
 
+    def listening(self, node, port=8083):
+        try:
+            cmd = "nc -z %s %s" % (node.account.hostname, port)
+            node.account.ssh_output(cmd, allow_fail=False)
+            self.logger.debug("Connect worker started accepting connections at: '%s:%s')", node.account.hostname, port)
+            return True
+        except (RemoteCommandError, ValueError) as e:
+            return False
+
+    # Currently the Connect worker supports waiting on three modes:
+    # INSTANT: return immediately
```

Comment:
if these constants aren't variables somewhere, then this should be a docstring instead of a comment above the method.